src/main/java/bio/terra/service/dataset/DatasetService.java

```diff
@@ -764,6 +724,18 @@ private static List<DatasetRequestAccessIncludeModel> getDefaultIncludes() {
         .collect(Collectors.toList());
   }
 
+  public static Column getColumn(Dataset dataset, String tableName, String columnName) {
```

Comment:
I thought this would be a good static method, since it doesn't rely on any `DatasetService` state. Please let me know if y'all have a different preferred approach.